dashboard/modules/reporter/reporter_agent.py

```diff
@@ -119,25 +119,25 @@ def jsonify_asdict(o) -> str:
         "node_gpus_available",
         "Total GPUs available on a ray node",
         "percentage",
-        ["ip", "SessionName"],
+        ["ip", "SessionName", "GpuDeviceName", "GpuIndex"],
```

Comment:
I am fine with that, but that will break the backward compatibility (since we documented these now... Not sure if we mentioned it is stable APIs). 

I also found "ip" is not needed because we use Instance labels

Reply:
I won't address it in this PR. Maybe I will make a follow up to use the same convention everywhere if we agree backward compatibility is not a concern.